config/sprayproxy.yaml

```
@@ -35,6 +35,24 @@ spec:
             limits:
               memory: "128Mi"
               cpu: "500m"
+        - name: kube-rbac-proxy
+          image: registry.redhat.io/openshift4/ose-kube-rbac-proxy:v4.12
+          args:
+            - "--secure-listen-address=0.0.0.0:8443"
+            - "--upstream=http://127.0.0.1:8080/"
```

Metrics port for the proxy is 6000, not 8080

```suggestion
            - "--upstream=http://127.0.0.1:6000/"
```

---

> Create a separate service for the metrics, sending traffic to port 8443

wrt that yeah @adambkaplan that was my original thought but then saw what was done with the original drop from @avinal you reviewed and assumed you all did that for a reason and built off of that.

otherwise sure working on the other bits now

---

oh and yeah looking at the build-service @adambkaplan there is no need for `containerPort: 6000`

I've removed that as well